### ISSUE TITLE
Opening and Closing transitions option no longer work with v5

### ISSUE THREAD

**Issue by laurent-d** [dimsemenov/PhotoSwipe#1882]
Seems the release of v5 breaks transition animations options :

Example : [https://photoswipe.com/opening-or-closing-transition/](https://photoswipe.com/opening-or-closing-transition/)

Have good day

**Comment by dimsemenov** [comment]
Works fine on my side, in what browser/os/device do you test?

**Comment by laurent-d** [comment]
Thanks @dimsemenov for the quick reply.

I tried on all browsers on my Mac Monterey M1 based : Chrome(V 100.0.4896.75) , Brave(V1.37.111), Safari (V 15.3) and Firefox (89.0.2 (64 bits))
I made a last one (just to be sure) on iPhone Safari transitions works very strange.

After check seems a settings on the Os Reduce animation > https://osxdaily.com/2018/12/17/how-reduce-motion-mac-disable-animations/ .. all is ok now .. It's the first time i encounter this type of effect. Usually it has no effects on websites

Sorry for the loss of time and the false alert
Once again sorry and thank you for your work.

Laurent
